ci/terraform/utils/bulk_user_email_dynamo_access.tf

```diff
@@ -0,0 +1,33 @@
+data "aws_dynamodb_table" "bulk_email_users" {
+  name = "${var.environment}-bulk-email-users"
```

Comment:
Do we need to add a count on this similar to in `dynamodb.tf`?

If we do, then we also need to update the terraform resources which reference this resource to use [0]